src/directives/decorators/stb-webmanual/stb-webmanual-decorator.js

```diff
@@ -30,6 +30,8 @@ angular.module('schemaForm').config(['schemaFormDecoratorsProvider', function(de
     successbox: base + 'successbox.html',
     errorbox: base + 'errorbox.html',
     infodate: base + 'date-info.html',
+    standarttable: base + 'standart-table.html',
+    addrowbutton: base + 'add-row-button.html',
```

Comment:
rename 'addrowbutton' to 'button' or 'actionButton' and do the same with name of file 'add-row-button.html' (e.g. button.html or action-button.html).